repo-transfer.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width">
+    <title>Transfer a repository to the W3C GitHub organization</title>
+    <link rel="stylesheet" href="css/wgio.css">
+    <link rel="icon" type="image/x-icon" href="//labs.w3.org/favicon.ico">
```

Comment:
```suggestion
```

Let's not have an official doc depends on the W3C labs server.

Reply:
addressed in https://github.com/w3c/w3c.github.io/pull/96/commits/9adbbf4151d916b4c398cfaf82d69d8c37de4bf4